draft-ietf-rats-corim.md

```diff
@@ -1259,6 +1259,22 @@ for appraisal until a CoBOM which activates that tag has been received and
 successfully processed. All the tags listed in the CoBOM must be activated in
 the same transaction, i.e., either all or none.
 
+The number of CoBOMs required in a given supply chain eco-system is dependent on
+Verifier Owner's appraisal policy, which is often driven by the complexity and nature of
+use case.
```

Comment:
```suggestion
The Verifier Owner's appraisal policy determines whether CoBOMs are required, this decision is made based on the complexity and nature of the supply chain eco-system.
```

Reply:
@andrew-draper I have taken good parts of your feedback and kept the conversation `use-case` driven.

Reply:
resolved